stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MediaAttachmentView.kt

```diff
@@ -139,7 +166,38 @@ internal class MediaAttachmentView : ConstraintLayout {
         }
     }
 
+    private fun Attachment.giphyInfo(field: GiphyInfoType): GiphyInfo? {
+        val giphyInfoMap = (extraData["giphy"] as Map<String, Any>?)?.get(field.value) as Map<String, String>?
+
+        return giphyInfoMap?.let { map ->
+            GiphyInfo(
+                url = map["url"] ?: this.thumbUrl ?: "",
+                width = map["width"]?.toInt() ?: GIPHY_INFO_DEAFULT_WIDTH,
+                height = map["height"]?.toInt() ?: GIPHY_INFO_DEAFULT_HEIGHT,
```

Comment:
Don't we need to use `dpToPx()` for the `GIPHY_INFO_DEAFULT_WIDTH` and `GIPHY_INFO_DEAFULT_HEIGHT`?